Lib/glyphsLib/types.py

```diff
@@ -362,6 +363,23 @@ def needsQuotes(string):
     return needsQuotes
 
 
+# FIXME: (jany) why is `feature_syntax_encode` different?
+def encode_dict_as_string_for_gsnode(value):
+    """Takes the PLIST string of a dict, and returns the same string
+    encoded such that it can be included in the string representation
+    of a GSNode."""
+    value = value.replace('"', '\\"')
+    return value.replace("\n", "\\n")
```

Comment:
@schriftgestalt There is an inconsistency here in string escaping between "\n" vs "\012", as noted in https://github.com/googlei18n/glyphsLib/issues/119. Do you still plan to fix it in Glyphs 2.5? If so, I can remove these two functions.